app/models/katello/docker_manifest.rb

```diff
@@ -2,7 +2,7 @@ module Katello
   class DockerManifest < Katello::Model
     include Concerns::PulpDatabaseUnit
 
-    has_one :docker_tag, :dependent => :destroy, :class_name => "Katello::DockerTag"
+    has_many :docker_tags, :dependent => :destroy, :class_name => "Katello::DockerTag"
```

Comment:
Probably the reason this page is not showing any tags -> http://i.imgur.com/lQWS3ug.png